format.sh

```diff
@@ -38,11 +38,22 @@ source venv/bin/activate
 
 echo "Checking Python files..."
 if [ "$COMMAND" == "fix" ]; then
-  black .
+  black --quiet .
 else
-  black --check --diff .
+  black --quiet --check --diff .
+fi
+
+echo "Running mypy type checks..."
+if ! mypy tests/parametric utils/parametric; then
```

Comment:
Just call mypy here, and configure it to look only in parametric in configuration file